### PR TITLE
Docs and UI improvement

### PR DESCRIPTION
**Pull request by geralddejong**

### CODE REVIEW

File: client/src/docs/command-docs.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import { Command } from "../state/app-state"
+
+export interface ICommandDoc {
+    title: Command,
+    body: JSX.Element
+}
+/*
+    Start = "Start",
+    Stop = "Stop",
+    Terraform = "Terraform",
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: ?

File: client/src/docs/command-docs.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            <CardText>
+                While you are fixing the island by terraforming (creating land or water),
+                you can decide to abandon the choice you made and start again.
+            </CardText>,
+    },
+    {
+        title: Command.ClaimHexalot,
+        body:
+            <CardText>
+                When you have reserved a hexalot, and fixed the island to make sure it follows
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: fixed=terraformed

File: client/src/docs/command-docs.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                If you don't like what you're seeing during the evolution and you've given up hope for
+                this particular gotchi, you can discard the current set of genes and start from scratch.
+            </CardText>,
+    },
+    {
+        title: Command.Evolve,
+        body:
+            <CardText>
+                You can start evolving either from the starting position hanging up in the tower
+                at the center of your hexalot, or you can ride a bit first and then start evolving.
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: Not clear what "riding" means here.

Comment (mvanderh) [reply client/src/docs/command-docs.tsx]: Just drop the sentence after the comma i'd say. Don't wanna make it TLDR

File: client/src/docs/command-docs.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                Evolving creates a group of idential galapagotchis which exist in the same space
+                at the same time, but have slightly mutated genes.
+                The slowest are forgotten and the new generation is made by mutating the genes
+                of the current crop of survivors.
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: Nice! Clear explanation. 

One thing though: they're not "identical"

File: client/src/docs/command-docs.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+                this particular gotchi, you can discard the current set of genes and start from scratch.
+            </CardText>,
+    },
+    {
+        title: Command.Evolve,
+        body:
+            <CardText>
+                You can start evolving either from the starting position hanging up in the tower
+                at the center of your hexalot, or you can ride a bit first and then start evolving.
+                Evolving creates a group of idential galapagotchis which exist in the same space
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]:
```suggestion
                Evolving creates a group of identical galapagotchis which exist in the same space
```

File: client/src/docs/command-docs.tsx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    title: Command,
+    body: JSX.Element
+}
+/*
+    Start = "Start",
+    Stop = "Stop",
+    Terraform = "Terraform",
+
+ */
+export const COMMAND_DOCS: ICommandDoc[] = [
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: You could simplify and make this a mapping `{[command: string]: JSXElement}`

File: client/src/docs/command-docs.tsx [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+    {
+        title: Command.AbandonFix,
+        body:
+            <CardText>
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: Also you should just return text, or a generic `span`/`div` if you need to. Don't wanna lock yourself into a `CardText`

File: client/src/docs/command-docs.tsx (hunk 1 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: oops.

File: client/src/docs/command-docs.tsx [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+                at the same time, but have slightly mutated genes.
+                The slowest are forgotten and the new generation is made by mutating the genes
+                of the current crop of survivors.
+            </CardText>,
+    },
+    {
+        title: Command.Home,
+        body:
+            <CardText>
+                Wherever you are, you can go back to your home position up in the tower in the
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: Maybe I'm not understanding, but why doesn't this just say "go back to your home position up in the tower.."? Don't you want to _just_ explain the command?

File: client/src/docs/command-docs.tsx [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+            <CardText>
+                While terraforming, you can choose to make the current hexagonal spot into water.
+            </CardText>,
+    },
+    {
+        title: Command.Plan,
+        body:
+            <CardText>
+                When you plan your journey, you zoom out and see the world from above.
+                Clicking on a new neighbor hexalot extends your journey one leg further,
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]:
```suggestion
                Clicking on a new neighbor hexalot adds a leg to your journey,
```

File: client/src/docs/command-docs.tsx [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+                and clicking on one that is already part of the makes the journey end there.
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]:
```suggestion
                and clicking on one that is already in the journey makes it end there.
```

File: client/src/docs/command-docs.tsx [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+            </CardText>,
+    },
+    {
+        title: Command.Ride,
+        body:
+            <CardText>
+                You can ride a gotchi, which means that it will be born from the center tower of its
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]: It's not clear why this is called "ride" then if you're just unleashing a gotchi to follow a journey

File: client/src/docs/command-docs.tsx (hunk 2 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: Yes!  changed that everywhere.

File: client/src/docs/command-docs.tsx [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+        body:
+            <CardText>
+                If you are tagging along behind a running gotchi, you can tell it to stop.
+            </CardText>,
+    },
+    {
+        title: Command.Terraform,
+        body:
+            <CardText>
+                After choosing a new hexalot, new spots appear at the edge of the island, but they begin
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]:
```suggestion
                After choosing an unoccupied hexalot, new spots appear at the edge of the island.
```

File: client/src/docs/command-docs.tsx [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+                not as land or water.  When you terraform, you decide.  The terraform button takes you to
```
Comment (mvanderh) [inline client/src/docs/command-docs.tsx]:
```suggestion
               you can terraform these spots to choose whether they'll be land or water. The terraform button takes you to
```

File: client/src/docs/toolbar-state-docs.tsx [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+    title: ToolbarState
+    body: JSX.Element
+}
+
+export const TOOLBAR_STATE_DOCS: IToolbarStateDoc[] = [
+    {
+        title: ToolbarState.AvailableSpot,
+        body:
+            <CardText>
+                This spot is one that you could claim if you wanted.
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]:
```suggestion
                This spot is unoccupied and can be claimed.
```

File: client/src/docs/command-docs.tsx (hunk 4 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: True

File: client/src/docs/toolbar-state-docs.tsx [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+            </CardText>,
+    },
+    {
+        title: ToolbarState.Planning,
+        body:
+            <CardText>
+                You are planning your journey.  This journey is where you galapagotchi rides when you let it go,
+                and it is the path that
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]: It seems you're still workin on this one

File: client/src/docs/toolbar-state-docs.tsx [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+            </CardText>,
+    },
+    {
+        title: ToolbarState.Evolving,
+        body:
+            <CardText>
+                Your gotchi is experiencing a kind of multiverse evolution.
+                It has been placed in competition with a number of mutations of itself, and they are
+                now locked in a fight for survival.
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]: Nice!

File: client/src/docs/command-docs.tsx (hunk 7 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: Sure but that's one find-replace.

File: client/src/docs/toolbar-state-docs.tsx [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+            </CardText>,
+    },
+    {
+        title: ToolbarState.Fixing,
+        body:
+            <CardText>
+                You have reserved a new hexalot, but this has created a bit of virgin territory at the edge
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]:
```suggestion
                You have reserved a new hexalot, but this has created virgin territory at the edge
```

File: client/src/docs/command-docs.tsx (hunk 8 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: Ok.

File: client/src/docs/toolbar-state-docs.tsx [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+            <CardText>
+                You have selected somebody else's galapagotchi.  You can take it for a ride to see how well
+                it can run.
+            </CardText>,
+    },
+    {
+        title: ToolbarState.FreeSpot,
+        body:
+            <CardText>
+                A free spot is a hexagonal spot which can be terraformed, which means that you can turn it
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]: You should stick to a single term for "a free/empty/virgin/unterraformed spot" or people will get confused

File: client/src/docs/toolbar-state-docs.tsx [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+                into land or water.
+            </CardText>,
+    },
+    {
+        title: ToolbarState.Home,
+        body:
+            <CardText>
+                You have selected your home hexalot, and from here you can plan your journey.
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]: Ain't it the gotchi's journey and not yours?

File: client/src/docs/command-docs.tsx (hunk 11 above)
Comment (geralddejong) [reply client/src/docs/command-docs.tsx]: Ok, what's a better word?

File: client/src/docs/toolbar-state-docs.tsx [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+                If you already have a journey planned you can try and ride it, or you can start the evolution
+                process so that your gotchi learns how to move around.
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]: Hmm, this is good but I don't like "move around", it's vague. Maybe walk? Develop motor skills?

File: client/src/docs/toolbar-state-docs.tsx [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+            <CardText>
+                You are a pioneer, so we can assume that you need no instructions.
+            </CardText>,
+    },
+    {
+        title: ToolbarState.Riding,
+        body:
+            <CardText>
+                You are riding on a gotchi, which means that you tag along close behind as the gotchi
+                runs each of the legs of its journey.
```
Comment (mvanderh) [inline client/src/docs/toolbar-state-docs.tsx]:
```suggestion
                runs each leg of its journey.
```

File: client/src/docs/toolbar-state-docs.tsx (hunk 19 above)
Comment (geralddejong) [reply client/src/docs/toolbar-state-docs.tsx]: Yes

File: client/src/docs/command-docs.tsx (hunk 11 above)
Comment (mvanderh) [reply client/src/docs/command-docs.tsx]: No it's a good word. I should clarify: what's unclear is why the subsequent act you describe is called "ride". Make clear to the user that their camera will be following the gotchi

**Pull request merged**